protocol-models/build.gradle

```diff
@@ -7,6 +7,8 @@ plugins {
 dependencies {
     implementation 'javax.validation:validation-api:1.1.0.Final'
     implementation 'org.apache.commons:commons-lang3:3.11'
+
+    implementation 'io.airbyte:airbyte-commons:0.40.32'
```

Comment:
we can't bring this dep along with us. whatever code we need from here, we should copy.

Reply:
ah ok.  Will we no longer have an `airbyte-commons` jar that we publish?

Reply:
So, LOL, this dependency. elasticsearch.common.collect.map. ANCIENT dependency that doesn't even exist in elasticsearch anymore. What was it used for? One single line in one test file. Changed to java.util.Map and it works just fine. Someone literally fat fingered in intellij probably to import the wrong Map

Reply:
Actually, this has the common json stuff too so we are probably going to have to copy some of it. Checking now